Bluewire.Common.GitWrapper/Parsing/StrictISO8601.cs

```diff
@@ -7,10 +7,16 @@ public class StrictISO8601
     {
         public const string Pattern = "yyyy-MM-ddTHH:mm:sszzzz";
 
+        private static readonly string[] patterns =
+        {
+            Pattern,

```

Comment:
is there a reason to keep Pattern as its own variable?

Reply:
It's public, so removing it is a breaking change.